packages/Button/src/Button.styles.js

```diff
@@ -52,19 +52,17 @@ const borderStyles = ({ kind, isDisabled, isActive }) => css`
 
 const focusStyles = css`
   &:focus {
-    outline: none;
+    ${stylers.focusRing()}
+
+    &:not(:focus-visible) {
+      box-shadow: none;
```

Comment:
@mikrotron could this be `${stylers.focusRing.none()}`?

Reply:
Where were you 7 days ago, before this was merged? You've changed, @alexzherdev 😆 

But yeah, that's a good suggestion for a new helper function that would give this code some very satisfying symmetry... 🤔 

cc @ShanAtDiligent